### ISSUE TITLE
news page on laptop screen

### ISSUE THREAD

**Issue by xmonader** [threefoldfoundation/www_threefold_io#349]
![image](https://github.com/threefoldfoundation/www_threefold_io/assets/64129/34796f90-375d-42b1-9b03-531cfa55a164)

this is how it looks

I think it gets fixed when i choose on of the filters on the right

![image](https://github.com/threefoldfoundation/www_threefold_io/assets/64129/db85d853-ad6a-406e-8ab1-a40db6a5e0f3)

**Comment by despiegk** [comment]
indeed have seen it on my screen too, + latest is not good its from 2021

**Comment by samtaggart** [comment]
@xmonader @despiegk Can you clarify the ask?

In the meantime @amandacaster Can you please make a spreadsheet of all of our news articles and the tags they have currently? We'll need to simplify the tagging system a bit to make sure that when people click on a tag, they don't end up with something from 2021.

**Comment by timurgordon** [comment]
Sure, on it @samtaggart

**Comment by samtaggart** [comment]
Blocked for now, waiting on script from Timur and clarification from @despiegk and @xmonader

**Comment by xmonader** [comment]
@samtaggart 

![image](https://github.com/threefoldfoundation/www_threefold_io/assets/64129/5df1bb6e-d0e1-4e8e-a460-88a20da23dd1)

this isn't usable at all, very weird spacing, not sure about the header too, but this screenshot isn't something i expect to see on any website.

when i clicked on one of  the filter, the view became more usable 


then the other problem is the listing of the news, the first item show is from 2021

**Comment by ehab-hassan** [comment]
@xmonader @samtaggart ll handel the ui here to make consistent and easy to use

**Comment by samtaggart** [comment]
@ehab-hassan Hi brother what is the status here?

**Comment by ehab-hassan** [comment]
@samtaggart fixed at the development
![image](https://github.com/threefoldfoundation/www_threefold_io/assets/10484295/5855a08b-ea36-4c48-bc05-453c9cb73bbe)

small screen

![image](https://github.com/threefoldfoundation/www_threefold_io/assets/10484295/398c2fb2-90ff-49e3-acda-c53714773115)

**Comment by samtaggart** [comment]
We have a new issue open on www_threefold_io to address the tagging / filtering part of this.

https://github.com/threefoldfoundation/www_threefold_io/issues/383
